src/ConnectedMode.UnitTests/Binding/UnintrusiveBindingPathProviderTests.cs

```diff
@@ -32,31 +30,30 @@ public class UnintrusiveBindingPathProviderTests
     {
         [TestMethod]
         public void MefCtor_CheckIsExported()
-        {
-            var serviceProvider = CreateConfiguredServiceProvider("");
+            =>  MefTestHelpers.CheckTypeCanBeImported<UnintrusiveBindingPathProvider, IUnintrusiveBindingPathProvider>(
+                    MefTestHelpers.CreateExport<ISolutionInfoProvider>());
 
-            MefTestHelpers.CheckTypeCanBeImported<UnintrusiveBindingPathProvider, IUnintrusiveBindingPathProvider>(
-                MefTestHelpers.CreateExport<SVsServiceProvider>(serviceProvider.Object),
-                MefTestHelpers.CreateExport<IThreadHandling>());
-        }
 
         [TestMethod]
-        public void Ctor_RunsOnUIThreadSync()
```

Comment:
Thread handling is now done the imported `ISolutionInfoProvider` so this test is redundant.